### PR TITLE
Bug 1748280: resourceapply: sync ConfigMap binaryData as well

### PR DESCRIPTION
**Pull request by stlaz**
We weren't syncing the binaryData which resulted in failures for the console operator. This fixes it.

### CODE REVIEW

**Comment by stlaz** [comment]
cc @mfojtik @jhadvig

File: pkg/operator/resource/resourceapply/core.go [hunk 1]
```diff
@@ -168,17 +169,28 @@ func ApplyConfigMap(client coreclientv1.ConfigMapsGetter, recorder events.Record
 			modifiedKeys = append(modifiedKeys, "data."+existingCopyKey)
 		}
 	}
+	for existingCopyKey, existingCopyBinValue := range existingCopy.BinaryData {
+		if requiredBinValue, ok := required.BinaryData[existingCopyKey]; !ok || !bytes.Equal(existingCopyBinValue, requiredBinValue) {
```
Comment (sttts) [inline pkg/operator/resource/resourceapply/core.go]: isn't binary data and data normalized to eachother by the apiserver?

Comment (jhadvig) [reply pkg/operator/resource/resourceapply/core.go]: no thats `stringData` in Secrets

**Pull request merged**